codersdk/agentsdk/logs.go

```diff
@@ -78,20 +78,20 @@ func (w *startupLogsWriter) Close() error {
 //
 // Neither Write nor Close is safe for concurrent use and must be used
 // by a single goroutine.
-func StartupLogsWriter(ctx context.Context, sender func(ctx context.Context, log ...StartupLog) error, level codersdk.LogLevel) io.WriteCloser {
+func StartupLogsWriter(ctx context.Context, sender func(ctx context.Context, log ...Log) error, level codersdk.LogLevel) io.WriteCloser {
```

Comment:
Should we rename this as well, `LogsWriter` and have it accept `source` in addition to level?

Reply:
Yup, will fix!